api/environments/identities/models.py

```diff
@@ -20,13 +22,75 @@
     )
 
 
+class DynamoIdentityManager:
```

Comment:
Can we extract all of this into a separate `dynamo` module? We should write some kind of wrapper for all of the dynamo logic there and then just instantiate the wrapper here. That goes for the writing to dynamo piece as well - it should all be done through the wrapper not this pseudo manager class.